inst/tutorials/treatment/treatment.Rmd

```diff
@@ -0,0 +1,203 @@
+
+```{r set-clin-treat-example, exercise = TRUE}
+
+# Establish a list of malariasimulation parameters:
+parameters <- get_parameters(overrides = list(human_population = 1000)) 
+
+# Load the parameters for artemether lumefantrine into the parameters
+parameters <- set_drugs(parameters = parameters, drugs = list(AL_params))
+
+# Specify a clinical treatment regime for artemether lumefantrine:
+parameters <- set_clinical_treatment(parameters = parameters, drug = 1, timesteps = c(0, (1 * 365) + 1), coverages = c(0, 0.6))
+
+# Tune the parameter set to an initial EIR:
+parameters <- set_equilibrium(parameters = parameters, init_EIR = 8)
+
+# Run the simulation
+clinical_treatment_simulation <- run_simulation(timesteps = 3 * 365, parameters = parameters)
+
+# Plot the prevalence through time:
+clinical_treatment_simulation |>
+  ggplot(aes(x = timestep, y = n_detect_lm_730_3650/n_730_3650)) +
+  geom_line(colour = "deepskyblue2", linewidth = 1.2) +
+  theme_bw() +
+  labs(x = "Time (days)", y = "Prevalence (2-10)")+
+  scale_y_continuous(limits = c(0, 1), expand = c(0, 0)) +
+  scale_x_continuous(expand = c(0, 0)) +
+  geom_vline(xintercept = (1 * 365) + 1, linetype = "dashed")
+  
+
+```
+
+```{r quiz2}
+
+quiz(
+  question("Drug parameters must be loaded into set_drugs() as a list?",
+    answer("TRUE", correct = TRUE),
+    answer("FALSE")
+  ),
+  question("If we want to specify clinical treatment regimes for three drugs, how many times must we use set_clinical_treatment()?",
+    answer("1", correct = TRUE),
+    answer("2"),
+    answer("3"),
+    answer("4")
+  ),
+   question("If we input the vector c(0, 365, 720) as the argument for timesteps into set_clinical_treatment(), which of the following vectors would NOT work for the coverages argument?",
+    answer("c(0.4, 0.5, 0.6)"),
+    answer("c(0, 0, 0)"),
+    answer("c(0.9, 0.4, 0.1)"),
+    answer("c(0, 0.5)", correct = TRUE)
+  )
+)
+
+```
+
+## 4: Putting it all together
+
+Awesome, that's all of the basics for running `malariasimulation` simulations with clinical treatment. For the final exercise, I want you to run two simulations to look at the effect of clinical treatment on malaria transmission dynamics.
+
+The first simulation is a baseline scenario with the following criteria:
+
+1.  `human_population` size of 10000
+2.  Equilibrate the parameters to an initial EIR of 8 (refer the "basics" tutorial if you need to!)
+3.  Run the simulation for 5 years
+
+The second simulation will use what we've learnt to simulate clinical treatment with the following criteria:
+
+1.  Set the `human_population` parameter to 10000
+2.  Load the parameters for dihydroartemisinin piperaquine (DHA-PQP)
+3.  Initiate clinical treatment with a coverage of 0.4 at the end of year 1
+4.  Increase the coverage to 0.8 at the end of the third year
+5.  Equilibrate the simulations with an initial EIR of 8
+6.  Run both simulations for 5 years
```

Comment:
Should 6 and 7 be a third section...? (Seems there's some redundancy)

Reply:
Have moved .7 to it's own line - wanted to keep the 5 years in each one for clarity